cypress/e2e/pages/viewApplication.cy.ts

```diff
@@ -69,4 +69,75 @@ describe('View a resident application', () => {
     ViewApplicationPage.getErrorSummary().should('be.visible');
     cy.contains(`Unable to delete household member`);
   });
+
+  it('shows the notes and history section for read only users', () => {
+    const application = generateApplication(applicationId, personId);
+    //ensure application requires assessment
+    application.status = ApplicationStatus.SUBMITTED;
+
+    cy.loginAsUser('readOnly');
+    cy.mockActivityHistoryApiEmptyResponse(applicationId);
+    cy.mockHousingRegisterApiGetApplications(applicationId, application);
+
+    ViewApplicationPage.visit(applicationId);
+    ViewApplicationPage.getNotesAndHistorySection().should('exist', {
+      timeout: 5000,
+    });
+  });
+
+  it('allows readonly users to see note section', () => {
+    const application = generateApplication(applicationId, personId);
+    //ensure application requires assessment
+    application.status = ApplicationStatus.SUBMITTED;
+
+    cy.loginAsUser('readOnly');
+    cy.mockActivityHistoryApiEmptyResponse(applicationId, true);
+    cy.mockHousingRegisterApiGetApplications(applicationId, application, true);
+    ViewApplicationPage.visit(applicationId);
+    ViewApplicationPage.getNotesAndHistorySection()
```

Comment:
Just thinking - as you're not writing in here anymore maybe enough to get it and make sure it exists?